auslib/web/common/swagger/releases_parameters.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
```

Comment:
Nothing special, it just seem more clean.
Do you think that is better in one file?